rotkehlchen/tests/api/test_addressbook.py

```diff
@@ -200,6 +200,40 @@ def test_get_addressbook(
     assert result['entries_found'] == 3
     assert result['entries_total'] == len(entries_set)
 
+    for attribute in ('name', 'address'):
+        # order by name/address in ascending order
+        response = requests.post(
+            api_url_for(
+                rotkehlchen_api_server,
+                'addressbookresource',
+                book_type=book_type,
+            ),
+            json={
+                'order_by_attributes': [attribute],
+                'ascending': [True],
+            },
+        )
+        result = assert_proper_response_with_result(response=response)
+        ascending_events = [AddressbookEntry.deserialize(data=raw_entry) for raw_entry in result['entries']]  # noqa: E501
+        assert len(ascending_events) > 0
+
+        # order by name/address in descending order
+        response = requests.post(
+            api_url_for(
+                rotkehlchen_api_server,
+                'addressbookresource',
+                book_type=book_type,
+            ),
+            json={
+                'order_by_attributes': [attribute],
+                'ascending': [False],
+            },
+        )
+        result = assert_proper_response_with_result(response=response)
+        descending_events = [AddressbookEntry.deserialize(data=raw_entry) for raw_entry in result['entries']]  # noqa: E501
+        descending_events.reverse()  # reverse the descending ordered events
+        assert descending_events == ascending_events  # order is ascending now
```

Comment:
you don't need to test that the logic for swapping the order works. This should be already tested since it is common logic what you are using